wurst/systems/modes/Respawn.wurst

```diff
@@ -80,6 +82,25 @@ init
                     timerDialog.display(display)
 
                     delayTimer.doAfter(delay) ->
-                        revive(dyingUnit, owner)
                         timerDialog.display(false)
                         timerDialog.destr()
+
+                        let units = ENUM_GROUP
+                            ..enumUnitsOfForce(owner.getAllies())
+                            .toLinkedList()
+                            ..sortWith((a, b) -> begin
+                                let aType = a.isType(UNIT_TYPE_STRUCTURE) ? 1 : 0
+                                let bType = a.isType(UNIT_TYPE_STRUCTURE) ? 1 : 0
+                                return aType == bType ?
```

Comment:
I'm not sure what your personal taste is, but I think it is generally recommended (and I prefer) to start both subsequent lines with the operator.

    return aType == bType
        ? a.getLevel() - b.getLevel()
        : aType - bType

Reply:
I'm ok with that